### ISSUE TITLE
all nodules at (0,0,0) shown in the result

### ISSUE THREAD

**Issue by Master-13** [juliandewit/kaggle_ndsb2017#6]
I run step3.py separately.

I downloaded the code and tried to run it on my own computer. (also with trained model and LUNA16 training data as test set) However, there are something wrong in the result --- all nodules are detected at (0,0,0) , and the "diameter_mm"s are all negative numbers.

I tried to debug step3.py and find something: at line 60-62, "center_x","center_y","center_z" equals to 0.0 no mater what the input image is.

How could I fix this problem? Waiting for your reply...

**Comment by juliandewit** [comment]
Line 62 is part of cleanup of predictions.

At row 282.
`p = model.predict(batch_data, batch_size=batch_size)`
What do you get in "p" ?

That one is important to validate if the model does something sensible.
The rest is just "bookkeeping".

**Comment by Master-13** [comment]
@juliandewit 
I made a simple mistake. Your code runs well with python3. In python2, "int/int" always equals 0 @line 294-296 in step3.py .

Thanks for your reply!

**Comment by juliandewit** [comment]
Great !
